src/ngx_http_lua_headers_out.c

```diff
@@ -106,6 +106,12 @@ static ngx_http_lua_set_header_t  ngx_http_lua_set_handlers[] = {
                  offsetof(ngx_http_headers_out_t, cache_control),
                  ngx_http_set_builtin_multi_header },
 
+#if defined(nginx_version) && nginx_version >= 1013009
```

Comment:
It should `nginx_version >= 1013010` rather than `1013009`.

`nginx_version` is always bumped after the last release, and the support of `Link` header is committed after the `1013009` bumped. Commits prior to this would also has this macro while cannot pass the compilation.

Reply:
Thanks, i am short on time right now and tried to grant you permissions on my fork. I didn't add tests yet because i never got feedback on my last post regarding tests.